notebook/lab/package.json

```diff
@@ -11,7 +11,7 @@
   },
   "devDependencies": {
     "css-loader": "^0.23.1",
-    "es6-promise": "^3.0.2",
+    "es6-promise": "^3.1.2",
```

Comment:
So we discussed this on gitter and my reasoning for `jupyter-js-widgets` was that only the file that was intended to be webpacked explicitly required `es6-promise`, and webpacking is a build-time thing.